budgetApp1/budgetApp1/BudgetViewController.swift

```diff
@@ -23,9 +23,7 @@ class BudgetViewController: UIViewController {
         super.viewDidLoad()
         
         viewModel = BudgetViewModel(repository: BudgetRepository(), delegate: self)
```

Comment:
The did swt on the viewModel is a bit strange. Can you nit instantiate the view model normally and then call the fillUI in the viewDidLoad?